src/mscorlib/shared/System/Diagnostics/Debug.cs

```diff
@@ -315,7 +345,8 @@ private sealed class DebugAssertException : Exception
         }
 
         // internal and not readonly so that the tests can swap this out.
-        internal static Action<string, string, string> s_ShowAssertDialog = ShowAssertDialog;
+        internal static Action<string, string, string, uint> s_ShowDialog = ShowDialog;
```

Comment:
This file is in the shared CoreLib partition. It is shared between CoreCLR and CoreRT/ProjectN. There will be matching change needed in ProjectN.

Reply:
I'll talk to Noah about making a matching change in ProjectN. Thanks for letting me know!